### PR TITLE
svelte 4 support

### PR DESCRIPTION
**Pull request by danhstevens**
Tested on svelte 4 and it seems to work perfectly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
 		"flatpickr": "^4.5.2"
 	},
 	"peerDependencies": {
-		"svelte": "^3.31.0"
+		"svelte": ">= 3.31.0 < 4.1"
```
Comment (schmidtk) [inline package.json:30]: Does this not work with `svelte >= 4.1`, or what was the reason for the restriction?

**Comment by schmidtk** [comment]
Based on the previous version pattern, might want to switch the `devDependencies` entry to `^4.0.0` and the package version to `4.0.0`. Is that accurate, @jacobmischka?

**Comment by bergstromdev** [comment]
Any updates on this?

**Comment by nicholasvperry** [comment]
+1

**Comment by CarlAmbroselli** [comment]
I've also tested it on web and mobile and looks like svelte-flatpickr has no issues with svelte 4.

**Comment by bighitbiker3** [comment]
Would also really love to get this merged!

**Comment by djavier-gigadat** [comment]
Tested in Svelte 4.2.0, with no issues. Waiting for this to be merged!

**Comment by ganigeorgiev** [comment]
@jacobmischka I know that a lot of people have spammed you about this PR and I apologies for doing the same, but I don't think the constraint suggested by @danhstevens is correct and there is really no reason to limit the peer dependency only up  to Svelte 4.1 (_there is already Svelte 4.2 available_).

In my opinion the dependency constraint need to change to:
- `>= 3.31.0 < 5.0.0`
- `>= 3.31.0 || ^4.0.0`

**Comment by jacobmischka** [comment]
Thanks for writing! I did that in master already in a separate commit before publishing: https://github.com/jacobmischka/svelte-flatpickr/blob/master/package.json#L29

**Comment by djavier-gigadat** [comment]
Hi @jacobmischka, any chance of when the new package version will be published with this change?

**Comment by ganigeorgiev** [comment]
_@djavier-gigadat I think it is already published in npm as [3.3.3 version](https://www.npmjs.com/package/svelte-flatpickr?activeTab=code)._

**Comment by djavier-gigadat** [comment]
Thanks @ganigeorgiev!

**Pull request merged**